caffe2/CMakeLists.txt

```diff
@@ -282,6 +282,8 @@ if(NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
     add_library(tensorpipe_agent "${TORCH_SRC_DIR}/csrc/distributed/rpc/tensorpipe_agent.cpp" "${TORCH_SRC_DIR}/csrc/distributed/rpc/tensorpipe_agent.h")
     target_link_libraries(tensorpipe_agent PRIVATE torch c10d tensorpipe fmt::fmt-header-only)
     add_dependencies(tensorpipe_agent torch c10d tensorpipe)
+    target_compile_definitions(tensorpipe_agent PRIVATE USE_TENSORPIPE)
+    target_compile_definitions(tensorpipe_agent PUBLIC USE_TENSORPIPE)
```

Comment:
Isn't the `PRIVATE` redundant with the following PUBLIC?

Reply:
Good point, fixed it.